src/app/pages/landing/landing.module.ts

```diff
@@ -2,16 +2,16 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { LandingComponent } from './landing/landing.component';
 import { SharedModule } from 'src/app/shared/shared.module';
-
-
+import { NewsletterModule } from '../newsletter/newsletter.module';
 
 @NgModule({
   declarations: [
     LandingComponent
   ],
   imports: [
     CommonModule,
-    SharedModule
+    SharedModule,
+    NewsletterModule
```

Comment:
newsletterul nu e o componenta asa de mare ... si ar trebui pusa in shared module...
nu are nevoie de propriul modul - pune componenta impreuna cu restul in shared module